### PR TITLE
API 7735 Table Populate Practitioner Role

### PR DESCRIPTION
**Pull request by that-guy-is-it**

### CODE REVIEW

**Comment by that-guy-is-it** [comment]
@nmoisan @jtaylor-va @haidong Pinging since this is a database update, populating the `Practitioner Role` column.

File: minimart-manager/src/test/java/gov/va/api/health/minimartmanager/minimart/MitreMinimartMaker.java [hunk 1]
```diff
@@ -470,6 +497,31 @@ private void insertByObservation(File file) {
     save(entity);
   }
 
+  @SneakyThrows
+  private void insertByPractitionerRole(File file) {
```
Comment (jhulbert-va) [inline minimart-manager/src/test/java/gov/va/api/health/minimartmanager/minimart/MitreMinimartMaker.java:501]: This method doesnt get used.

File: minimart-manager/pom.xml [hunk 2]
```diff
@@ -13,7 +13,7 @@
   <properties>
     <datamart-starter.version>3.0.0</datamart-starter.version>
     <apache-csv.version>1.8</apache-csv.version>
-    <data-query.version>3.0.192</data-query.version>
+    <data-query.version>3.0.202-SNAPSHOT</data-query.version>
```
Comment (jhulbert-va) [inline minimart-manager/pom.xml:16]: Shoudlnt be a snapshot value

File: minimart-manager/src/test/java/gov/va/api/health/minimartmanager/minimart/MitreMinimartMaker.java (hunk 1 above)
Comment (that-guy-is-it) [reply minimart-manager/src/test/java/gov/va/api/health/minimartmanager/minimart/MitreMinimartMaker.java:501]: Good catch, must have been leftover from a refactor.

File: minimart-manager/pom.xml (hunk 2 above)
Comment (that-guy-is-it) [reply minimart-manager/pom.xml:16]: This will get updated shortly, after the current DQ PR merges in and the non-snapshot version becomes available.

**Review by jblefkove-va — APPROVED** [review]

**Pull request merged**